ngx-fudis/projects/ngx-fudis/src/lib/components/breadcrumbs/breadcrumbs.component.html

```diff
@@ -0,0 +1,11 @@
+<nav aria-label="Breadcrumb navigation" class="fudis-breadcrumbs" *ngIf="links.length > 0">
```

Comment:
Change this aria-label to be required Input instead of hard-coded value. 
We want to be able to modify the label in order to tell the screen reader which breadcrumb is being navigated e.g in different sections of the page and that the breadcrumbs are in correct user language.